src/OptionsPopup.qml

```diff
@@ -29,414 +27,431 @@ Popup {
     property string cloudinitwrite
     property string cloudinitnetwork
 
-    // background of title
```

Comment:
Ditto for this file, use the Github.dev editor to view the diff and you'll see that the changes are much smaller.